### ISSUE TITLE
[F.Rq.] TAStudio manual restore

### ISSUE THREAD

**Issue by YoshiRulz** [TASEmulators/BizHawk#3556]
like auto-restore but triggered by a hotkey
adelikat says:
> restore hotkey, will require always calculating the restore point (currently only does if auto-restore is on)
> pressing hotkey will perform the restore logic auto-restore would have
> useful for pressing multiple buttons before needing to see the restore results

**Comment by vadosnaprimer** [comment]
Middle mouse button.

**Comment by tommai78101** [comment]
@vadosnaprimer Rip MacBook trackpad users

**Comment by vadosnaprimer** [comment]
Not saying it shouldn't be configurable.

**Comment by YoshiRulz** [comment]
Three-finger MMB works on my Ubuntu laptop.

**Comment by vadosnaprimer** [comment]
Yeah that's not good. I guess it's the case when RMB+wheel can't really be used either, so a configurable key is indeed required.

**Comment by SuuperW** [comment]
Manual restore is a hotkey now. There are also some fixes for menu access in a new PR by me, #4480. There are some other things that appear impossible without a mouse:
- selecting any non-menu controls, and therefore pretty much everything
- opening the right-click menu for the input roll, and therefore
-- ~~cancel seek~~ (done)
-- start new project from saveram (very strange to have this in the context menu instead of the file menu)
